runtime/runtime-core/common/test/aws/smithy/kotlin/runtime/net/url/QueryParametersTest.kt

```diff
@@ -22,25 +22,25 @@ class QueryParametersTest {
     fun testDecodedParametersAlternateEncoding() {
         val paramString = QueryParameters {
             decodedParameters {
-                add("a", "green apple")
```

Comment:
Should we not retain testing for space handling difference as well as `:`?

Reply:
Can do!